Solutions/Forescout Host Property Monitor/Playbooks/ReadMe.md

```diff
@@ -0,0 +1,68 @@
+
+<a name="playbooks">
+
+# Playbook Templates
+This package includes:  
+
+* DNS Sniff Event Playbook   
+This topic describes a sample Logic App/Playbook, "DNS Sniff Event Analysis".
+Use the Analytic Rule to check what kind of incident is created and how many times the events happened for any endpoint within the time interval. Based on this information, the Logic App/Playbook will set the Description field of the incident with the following information:  
+  -	action_group: Indicates the category of action. The valid values are manage, notify, remediate, and restrict.  
+  -	disposition: Represents the actual action in 2-digit format. For example, 00 for Recheck Host, 10 for HTTP Notification.  
+  -	endpoint_ip: Indicates the IP address of the endpoint, which can be an IPv4 or IPv6 address.  
+  -	em_ip: Indicates the IP address of the Enterprise Manager to which the endpoint belongs.  
+  -	info: Indicates addistional information   
+The em_ip and info are optional.  
+
+Note that the eyeExtend for Microsoft Sentinel module polls the incidents information periodically and examines the Description field of each incident. Based on the information set in the Description field, the Continuum Platform applies the requested action set by Microsoft Sentinel to the endpoints.  
+The following is a sample description field of an incident set by the playbook:  
+action_group="restrict";disposition="31";endpoint_ip=["10.16.141.130"];em_ip=["10.16.147.96"];info=Generated_by_Dnsniff_event  
+Each action has a unique disposition value. The action name, disposition value, and action group supported by eyeExtend for Microsoft Sentinel are as follows:  
+
+Recheck Host	00	manage  
+HTTP Notification	10	notify  
+HTTP Redirection to URL	11	notify  
+Send Balloon Notification	12	notify  
+Send Email	13	notify  
+Send Email to User	14	notify  
+MAC User Notification	15	notify  
+Disable Dual Homed	20	remediate  
+Kill Cloud Storage	21	remediate  
+Kill Instant Messaging	22	remediate  
+Kill Peer-to-peer	23	remediate  
+Switch Block	30	restrict  
+Endpoint Address ACL	31	restrict  
+Disable External Device	32	restrict  
+Expedite IP Discovery	33	restrict  
+Get Microsoft SMS/SCCM Updates	34	restrict  
+Start Windows Updates	35	restrict  
+Start Macintosh Updates	36	restrict  
+Windows Self Remediation	37	restrict  
+Block Network Access	38	restrict  
+Block Suspected MAC Spoofing	39	restrict  
+RADIUS Authorize	40	restrict  
+VPN Block	41	restrict  
+WLAN Block	42	restrict  
+Whitelist Network Access	43	restrict  
+Use the Microsoft Apps Designer to view or edit the playbook as needed to fulfil security requirements. The code is prepared by using the visual editor, which can be easily viewed and modified.  
+
+<a name="limitations">
+  
+# Known issues and limitations  
```

Comment:
Since this section does not have any information, we can remove it.

Reply:
@devikamehra  Thanks for the review. Updated files according to review comments. Thanks.